### PR TITLE
Simple streaming example. 

### PR DESCRIPTION
**Pull request by mfirry**
Simple streaming example using `firehoseStatuses` api.

### CODE REVIEW

**Comment by DanielaSfregola** [comment]
Hi @mfirry,
 thank you very much for your contribution!

Looking at the PR now...

File: src/main/scala/streaming/SampleStatusesHashtagPrinter.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import com.danielasfregola.twitter4s.TwitterStreamingClient
+import com.danielasfregola.twitter4s.entities.Tweet
+
+
+object SampleStatusesHashtagPrinter extends App {
+
+  val client = TwitterStreamingClient()
+
+  def printHashtags(tweet: Tweet) = tweet.entities match {
```
Comment (DanielaSfregola) [inline src/main/scala/streaming/SampleStatusesHashtagPrinter.scala:11]: Function not used anywhere. Is this intentional?

Comment (mfirry) [reply src/main/scala/streaming/SampleStatusesHashtagPrinter.scala:11]: Yeah... well now it is :D

**Comment by DanielaSfregola** [comment]
LGTM, thanks for your contribution @mfirry!

**Pull request merged**